### PR TITLE
Add Code of Conduct

### PR DESCRIPTION
**Pull request by jrfnl**
.. which will automatically apply to all projects within this organisation.

👉🏻 **TODO**: **we need to discuss which email address to add to report violations.**

Refs:
* https://www.contributor-covenant.org
* https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at [TODO: add email address]
```
Comment (grogy) [inline CODE_OF_CONDUCT.md]: I suggest add both e-mails. My (vaclav@makes.cz) and yours. Will be choice of reporter.

**Review by grogy — APPROVED** [review]

**Comment by grogy** [comment]
Thank you. I read the whole page and it's makes sense.

I am sorry that something like this can happen and I trust that it will not need to be addressed in our area.

**Comment by jrfnl** [comment]
I've updated the PR to include both our email addresses, as per your suggestion.

**Comment by grogy** [comment]
Thank you, I merged it.

**Pull request merged**